css/layout.css

```diff
@@ -1,19 +1,26 @@
+html {
```

Comment:
Typically you will use a universal selector ( `*)` for this and put it at the top of your base.css (above the reset)